forge-gui/res/cardsfolder/upcoming/surge_engine.txt

```diff
@@ -0,0 +1,10 @@
+Name:Surge Engine
+ManaCost:2
+Types:Artifact Creature Construct
+PT:3/2
+K:Defender
+A:AB$ Animate | Cost$ U | HiddenKeywords$ Unblockable | RemoveKeywords$ Defender | Duration$ Permanent | SpellDescription$ CARDNAME loses defender and gains "This creature can’t be blocked."
```

Comment:
`’`

Reply:
Please make this a static instead of a keyword... see #1551 

The engine already supports this, the PR is just where we'll make the changeover for old cards